docker-compose.yml

```diff
@@ -18,8 +18,22 @@ services:
         image: vencord/backend
         build: .
         restart: always
-        env_file:
-            - .env
         depends_on:
             redis:
                 condition: service_healthy
+        environment:
+            PORT: 8080
+            HOST: ${HOST}
+            REDIS_URI: ${REDIS_URI}
+            ROOT_REDIRECT: ${ROOT_REDIRECT}
+            DISCORD_CLIENT_ID: ${DISCORD_CLIENT_ID}
+            DISCORD_CLIENT_SECRET: ${DISCORD_CLIENT_SECRET}
+            DISCORD_REDIRECT_URI: ${DISCORD_REDIRECT_URI}
+            PEPPER_SECRETS: ${PEPPER_SECRETS}
+            PEPPER_SETTINGS: ${PEPPER_SETTINGS}
+            SIZE_LIMIT: ${SIZE_LIMIT}
+            ALLOWED_USERS: ${ALLOWED_USERS}
+            PROMETHEUS: ${PROMETHEUS}
+            PROXY_HEADER: ${PROXY_HEADER}
```

Comment:
wdym? I'm only talking about changing the way you do environment variables here

Reply:
Ah okay, I misunderstood that part then.